packages/actor-http-retry/lib/ActorHttpRetry.ts

```diff
@@ -222,16 +222,13 @@ export class ActorHttpRetry extends ActorHttp {
   }
 }
 
-export interface IActorHttpQueueArgs extends IActorHttpArgs {
+export interface IActorHttpRetryArgs extends IActorHttpArgs {
```

Comment:
The name of this interface was supposed to mirror the actor name, but I forgot to change it apparently. So I included this small change here because I only noticed it now. The actor was called 'queue' during development.